### ISSUE TITLE
Extra spaces included in manuscript chapter wordcount

### ISSUE THREAD

**Issue by gfishbone** [worldanvil/worldanvil-bug-tracker#98]
**World Anvil Username**: gfishbone

**World / Campaign / RPG Character / Manuscript Title**: Any manuscript

**Feature**:  Word Count

**Related URLS** (To public & visible pages add screenshots otherwise):

**Describe the Issue**
When multiple spaces are used for formatting in a scene, the spaces are rendered as alternating space and non-breaking space characters, which are being incorrectly included in chapter word counts. Word counts in the editor word counts are correct.

**To Reproduce**
1. Go to any manuscript.
2. Create a text scene.
3. Type some number of spaces but no words.
4. See that the word counter in the editor registers 0 words.
5. Click on the chapter containing that scene.
6. See that some number of words have been counted in error.

**Expected behavior**
&nbsp; characters should not register as words. The word count in the chapter view should match the word count in the scene edit screen.

**Screenshots**
Correct Word Count (200) in Editor:

![image](https://user-images.githubusercontent.com/1477119/112411722-c2519480-8cf3-11eb-89bc-9c6ad5d310b8.png)

Incorrect Word Count (277) for same scene in Chapter View:

![image](https://user-images.githubusercontent.com/1477119/112411909-02b11280-8cf4-11eb-895a-304c6e9a9bb6.png)

**Comment by iamromeo** [comment]
This is an already submitted bug, in fact by you I think @gfishbone 
The priority because of the uniqueness of the usercase is set to lowest

**Comment by gfishbone** [comment]
I submitted this months ago in the former bug reporting system and hadn't heard anything, and the bug persists. It renders the wordcount function unusable for me. I couldn't find it in a search on this new bug tracker, so assumed it had been lost in the shuffle. Not sure why it's set to "lowest" priority -- I encounter it literally every day and it drives me bonkers. I have had to keep track of my wordcount off-site in a spreadsheet. 

An accurate wordcount is of HIGH importance, as manuscripts are defined by their length and an accurate count is necessary to track progress and to plan the overall structure of the story. When writing a verse novel, multiple spaces are required in every stanza of every scene--there is no other way to present the proper formatting. The lack of an accurate wordcount is a major challenge in planning and preparing my content every day in every chapter I write.
